<launch>
    <arg name="image_width" default="1280"/>
    <arg name="image_height" default="720"/>
    <arg name="frame_rate" default="20.0"/>
    <arg name="pixel_format" default="mjpeg" />

    <node name="right_camera" pkg="usb_cam" type="usb_cam_node">
        <param name="video_device" value="/dev/video0"/>
        <param name="image_width" value="$(arg image_width)"/>
        <param name="image_height" value="$(arg image_height)"/>
        <param name="pixel_format" value="$(arg pixel_format)"/>
        <param name="camera_name" value="right_camera"/>
        <param name="frame_rate" value="$(arg frame_rate)"/>
    </node>
</launch>
